src/main/java/CashDrawer.java

```diff
@@ -70,6 +70,22 @@ public void setTotal() {
         this.total = (20 * twenty)  + (10 * ten) + (5 * five) + (2 * two) + (1 * one);
     }
 
+    public void putBillsInRegister(int twenty, int ten, int five, int two, int one){
```

Comment:
Same question as on the original, do we need to validate these inputs, as you can have negative integers here.